src/Core/Settings.h

```diff
@@ -104,6 +104,7 @@ class IColumn;
     M(Bool, s3_check_objects_after_upload, false, "Check each uploaded object to s3 with head request to be sure that upload was successful", 0) \
     M(Bool, s3_allow_parallel_part_upload, true, "Use multiple threads for s3 multipart upload. It may lead to slightly higher memory usage", 0) \
     M(Bool, s3_throw_on_zero_files_match, false, "Throw an error, when ListObjects request cannot match any files", 0) \
+    M(Bool, s3_disable_checksum, false, "Disable checksums for S3 to avoid excessive reads", 0) \
```

Comment:
The documentation is not extensive enough. If a user reads it, it might think that we don't have checksums. Also, it confusingly says that it is related to reads, while in fact, it is related to writes.